.github/workflows/integration-tests.yml

```
@@ -105,5 +105,5 @@ jobs:
           wget --quiet ${CARTOPY_FEATURE}
           mkdir -p ${CARTOPY_SHARE_DIR}
           python cartopy_feature_download.py physical --output ${CARTOPY_SHARE_DIR} --no-warn
-      - run: pytest -W error -W "ignore:numpy.ndarray size changed:RuntimeWarning"
+      - run: pytest -m "not image"
```

Thanks, I agree with removing the Warning option, but we should not add the "not image" option in my mind. Let's keep CI error and that will be the driving force to resolve https://github.com/bjlittle/geovista/issues/447. (Sorry for the delay in resolving the issue.)
```suggestion
      - run: pytest
```